mne_bids_pipeline/_io.py

```diff
@@ -6,3 +6,13 @@
 def _write_json(fname, data):
     with open(fname, 'w') as f:
         json_tricks.dump(data, fp=f, allow_nan=True, sort_keys=False)
+
+
+def _read_json(fname):
+    with open(fname, 'r') as f:
+        return json_tricks.load(f)
+
+
+def _empty_room_match_path(run_path, cfg):
```

Comment:
I know we don't do this consistently with existing code, but could you please add type annotations to the function signatures, both for input and return values? Just for these helpers here in this module, not for the actual processing step script

Reply:
Done, feel free to take a look to see if I did it correctly. I'm still a type annotation novice